### PR TITLE
Adds sensible runner keybind defaults

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Please more than feel free to comment your own keybindings or suggest tweaks, see the file tab for added defaults

## Why It's Good For The Game

For ease of entering for new players into the caste

## Changelog
:cl:
qol: Added runner keybind defaults
/:cl:

### CODE REVIEW

**Comment by Lefunny** [comment]
image

Runner based binds

**Comment by yyzsong** [comment]
Evade should be space, honestly. Good for emergency abilities.

**Comment by Joe13413** [comment]
> Evade should be space, honestly. Good for emergency abilities.

personal preference honestly. I use E for a lot of my evasion/getting the fuck outta somewhere abilities. At least we got keybinds for newer players now

**Comment by Lefunny** [comment]
> Evade should be space, honestly. Good for emergency abilities.

Space is hide for when you pounce marines

**Comment by Joe13413** [comment]
> > Evade should be space, honestly. Good for emergency abilities.
> 
> Space is hide for when you pounce marines

thats what i use space for yeh

File: code/datums/keybinding/xeno.dm [hunk 1]
```diff
@@ -596,12 +596,14 @@
 	full_name = "Runner: Evasion"
 	description = "Take evasive action, forcing non-friendly projectiles that would hit you to miss so long as you keep moving."
 	keybind_signal = COMSIG_XENOABILITY_EVASION
+	hotkey_keys = list("E")
```
Comment (TiviPlus) [inline code/datums/keybinding/xeno.dm:600]:
```suggestion
	hotkey_keys = list("Space")

```

**Pull request merged**